File: android/src/main/kotlin/io/openim/flutter_openim_sdk_ffi/models/PublicUserInfo.kt
package io.openim.flutter_openim_sdk_ffi.models

class PublicUserInfo {
    /**
     * 用户id
     */
    var userID: String? = null

    /**
     * 昵称
     */
    var nickname: String? = null

    /**
     * 头像
     */
    var faceURL: String? = null

    /**
     * 性别
     */
    var gender = 0

    /**
     *
     */
    var appMangerLevel = 0

    /**
     * 扩展字段
     */
    var ex: String? = null
}

File: ios/Classes/models/PublicUserInfo.swift
public class PublicUserInfo: Decodable {
    /**
     * 用户id
     */
    var userID: String?
    
    /**
     * 昵称
     */
    var nickname: String?
    
    /**
     * 头像
     */
    var faceURL: String?
    
    /**
     * 性别
     */
    var gender = 0
    
    /**
     *
     */
    var appMangerLevel = 0
    
    /**
     * 扩展字段
     */
    var ex: String?
}
